services/QuillCMS/Gemfile

```diff
@@ -8,8 +8,9 @@ git_source(:github) do |repo_name|
 end
 
 # CORE
-gem 'puma', '~> 3.12.6'
+gem 'puma', '~> 4.3.8'
 gem 'rails', '~> 6.1'
+gem "actionpack", ">= 6.1.3.2"
```

Comment:
I think if you do `bundle update rails` then the `actionpack` dependency is unnecessary since the `rails` gem keeps all of the railties on the same version.  A quick diff on Gemfile.lock would confirm this.

Reply:
Done